src/EnergyPlus/DXCoils.cc

```diff
@@ -10407,9 +10409,12 @@ Label50: ;
 		Real64 OutletAirHumRat; // outlet air humidity ratio [kg/kg]
 		Real64 TotCapHS; // total capacity at high speed [W]
```

Comment:
As best I can tell, since I'm not sure what power would have been wo defrost, that plot does appear to show that power is now slightly higher during defrost which makes sense. I imagine the wo defrost power line would split the two. Since capacity didn't change, I guess there is no reason to plot.